astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,447 @@
+# 1. standard library imports
+
+from collections import OrderedDict
+import re
+import warnings
+
+# 2. third party imports
+from astropy.time import Time
+from astropy import table
+from astropy.io import ascii
+from bs4 import BeautifulSoup
+
+# 3. local imports - use relative imports
+# commonly required local imports shown below as example
+# all Query classes should inherit from BaseQuery.
+from ...query import BaseQuery
+from ...utils import async_to_sync
+
+# import configurable items declared in __init__.py, e.g. hardcoded dictionaries
+from . import conf
+
+
+@async_to_sync
+class RingNodeClass(BaseQuery):
+    """
+    for querying the Planetary Ring Node ephemeris tools
+    <https://pds-rings.seti.org/tools/>
+
+
+    # basic query for all six targets
+    for major_body in ['mars', 'jupiter', 'uranus', 'saturn', 'neptune', 'pluto']:
+        nodequery = RingNode(major_body, '2022-05-03 00:00')
+        systemtable, bodytable, ringtable = nodequery.ephemeris()
+
+        print(' ')
+        print(' ')
+        print('~'*40)
+        print(major_body)
+        print('~'*40)
+        print(systemtable)
+        print(bodytable)
+        print(ringtable)
+
+
+    """
+
+    TIMEOUT = conf.timeout
+
+    def __init__(self, planet=None, obs_time=None):
+        """Instantiate Planetary Ring Node query
+
+        Parameters
+        ----------
+        planet : str, required. one of Jupiter, Saturn, Uranus, or Neptune
+            Name, number, or designation of the object to be queried.
+        obs_time : str, in JD or MJD format. If no obs_time is provided, the current
```

Comment:
Time should use an astropy `Time` object.  This will facilitate transformations between time formats and scales.

Reply:
is it ok/good to make it so that either an astropy Time object or a string can be passed? and then within the code if a string is passed then it becomes a Time object?

Reply:
I think that is OK, but you will need to document the time scale assumed.